src/requesthandler/RequestHandler_General.cpp

```diff
@@ -92,6 +96,42 @@ RequestResult RequestHandler::GetStats(const Request&)
 	return RequestResult::Success(responseData);
 }
 
+/**
+ * Gets a list of connected monitors and information about them.
+ *
+ * @responseField monitors | Array<Object> | a list of detected monitors with some information
+ *
+ * @requestType GetMonitorList
+ * @complexity 2
+ * @rpcVersion -1
+ * @initialVersion 5.0.0
+ * @category general
```

Comment:
This would be better placed in the Ui category

Reply:
moved to UI caegory